src/libtiled/varianttomapconverter.cpp

```diff
@@ -966,7 +981,12 @@ bool VariantToMapConverter::readTileLayerData(TileLayer &tileLayer,
 Properties VariantToMapConverter::extractProperties(const QVariantMap &variantMap) const
 {
     return toProperties(variantMap[QStringLiteral("properties")],
-                        variantMap[QStringLiteral("propertytypes")]);
+            variantMap[QStringLiteral("propertytypes")]);
```

Comment:
```suggestion
                        variantMap[QStringLiteral("propertytypes")]);
```
Please leave the existing indentation style intact. :-)

Reply:
oops